pages/profile/read/views/index.jsx

```diff
@@ -9,7 +9,7 @@ const Index = ({
 }) => {
 
   return (
-    <Profile establishment={establishment} profile={model} allowedActions={allowedActions} title={establishment.name}/>
+    <Profile title={model.name} profile={model} establishment={establishment} estName={establishment.name} allowedActions={allowedActions} />
```

Comment:
I would add the profile name in an `h1` here instead of inside the `<Profile />` component.

Reply:
done